resources/istio-kyma-patch/templates/configmap.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio-kyma-patch-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: istio-kyma-patch
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+data:
+  istio-egressgateway.deployment.patch.json: |
+    [
+      {
+        "op": "replace",
+        "path": "/spec/template/spec/containers/0/args/15",
+        "value": "zipkin.kyma-system:9411"
+      }
+    ]
+
+  istio-ingressgateway.deployment.patch.json: |
+    [
+      {
+        "op": "replace",
+        "path": "/spec/template/spec/containers/0/args/15",
+        "value": "zipkin.kyma-system:9411"
+      }
+      {{ if .Values.global.isLocalEnv }}
+      ,{
+        "op": "add",
+        "path": "/spec/template/spec/containers/0/ports/0/hostPort",
+        "value": 80
+      },{
+        "op": "add",
+        "path": "/spec/template/spec/containers/0/ports/1/hostPort",
+        "value": 443
+      }
+      {{ end }}
+    ]
+
+  istio-pilot.deployment.patch.json: |
+    [
+      {
+        "op": "add",
+        "path": "/spec/template/spec/containers/0/args",
+        "value": ["discovery", "--webhookEndpoint", "http://istio-webhook:5000"]
+      }
+    ]
+
+  istio-policy.deployment.patch.json: |
+    [
+      {
+        "op": "replace",
+        "path": "/spec/template/spec/containers/0/args/4",
+        "value": "--trace_zipkin_url=http://zipkin.kyma-system:9411/api/v1/spans"
+      }
+    ]
+
+  istio-telemetry.deployment.patch.json: |
+    [
+      {
+        "op": "replace",
+        "path": "/spec/template/spec/containers/0/args/4",
+        "value": "--trace_zipkin_url=http://zipkin.kyma-system:9411/api/v1/spans"
+      }
+    ]
+
+
+  delete: |
+    deployment prometheus
+    deployment istio-tracing
+    deployment servicegraph
+    deployment grafana
+    job istio-grafana-post-install
+    service grafana
+    service jaeger-agent
```

Comment:
Why are we deleting Jaeger components ?

Reply:
It's istio jeager deleted from `istio-system` namespace. Previously we had it disabled, but now we cannot control whether it's installed or not. As we introduce our own jaeger it ahs to be removed from istio.